internal/plugin.go

```diff
@@ -598,10 +609,17 @@ func (p *Plugin) assertValidConfig() error {
 			"policyDefaults must provide only one of placement.labelSelector or placement.clusterSelectors",
 		)
 	}
+	if p.PolicyDefaults.Placement.PlacementRuleName != "" && p.PolicyDefaults.Placement.PlacementName != "" {
+		return errors.New(
+			"policyDefaults must provide only one of placement.placementName or placement.placementRuleName",
+		)
+	}
+
 	if (len(p.PolicyDefaults.Placement.LabelSelector) != 0 || len(p.PolicyDefaults.Placement.ClusterSelectors) != 0) &&
-		(p.PolicyDefaults.Placement.PlacementRulePath != "" || p.PolicyDefaults.Placement.PlacementPath != "") {
+		((p.PolicyDefaults.Placement.PlacementRulePath != "" || p.PolicyDefaults.Placement.PlacementPath != "") ||
```

Comment:
This should also account for when the placement path and placement name are both set.